parts/linux/cloud-init/artifacts/components.json

```diff
@@ -784,36 +784,36 @@
             "versionsV2": [
               {
                 "k8sVersion": "1.27",
-                "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.27.101-akslts",
-                "previousLatestVersion": "1.27.100-akslts"
+                "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/kubernetes-node",
```

Comment:
What is OCI_registry ?

Reply:
Setting a new keyword to distinguish from the containerImage one `registry`. OCI_registry here means it's a registry hosting OCI artifacts. I apply different package rule to this keyword.

Reply:
oh. You are right. I forgot to paste the OCI_registry custom manager from my fork. Thanks for pointing this out. Added it back